cmd/deploy/start.go

```diff
@@ -144,7 +144,7 @@ func start(cmd *cobra.Command, configuration *config.Configuration, options *dep
 	if options.pipelineID != "" {
 		options.deploymentFile =
 			fmt.Sprintf("armory::%s/pipelines/%s/config", configuration.GetArmoryCloudAddr().String(), options.pipelineID)
-		withConfiguration = WithPipelineId
+		withConfiguration = WithURL
```

Comment:
i like this. from user perspective - it makes no difference what kind of url it is.